internal/service/rds/cluster_activity_stream.go

```diff
@@ -0,0 +1,164 @@
+package rds
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	"github.com/hashicorp/terraform-provider-aws/internal/verify"
+)
+
+func ResourceClusterActivityStream() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAwsRDSClusterActivityStreamCreate,
+		ReadContext:   resourceAwsRDSClusterActivityStreamRead,
+		DeleteContext: resourceAwsRDSClusterActivityStreamDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"resource_arn": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: verify.ValidARN,
+			},
+			"kms_key_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"mode": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					rds.ActivityStreamModeSync,
+					rds.ActivityStreamModeAsync,
+				}, false),
+			},
+			"kinesis_stream_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceAwsRDSClusterActivityStreamCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	conn := meta.(*conns.AWSClient).RDSConn
+
+	resourceArn := d.Get("resource_arn").(string)
+	kmsKeyId := d.Get("kms_key_id").(string)
+	mode := d.Get("mode").(string)
+
+	startActivityStreamInput := &rds.StartActivityStreamInput{
+		ResourceArn:      aws.String(resourceArn),
+		ApplyImmediately: aws.Bool(true),
+		KmsKeyId:         aws.String(kmsKeyId),
+		Mode:             aws.String(mode),
+	}
```

Comment:
move these inline. for example `Mode:             aws.String(d.Get("mode").(string)),`  unless there is any reason not to do so, prefer inline.

Reply:
Done. Left `resourceArn` because it's referenced later in the function.

Reply:
i think you mightve not pushed the changes?

Reply:
@DrFaust92 you're right. I marked these resolved before pushing. I'm having trouble with the Acceptance Tests. Do you know how to debug an acceptance test using Visual Studio Code? The test keeps exiting with no error before I'm able to jump into code.